src/Settings/Mods.ts

```diff
@@ -309,9 +309,13 @@ export function GenerateModsSettings(): GameStartr.IModAttachrCustoms {
                         for (const actor of opponent.actors) {
                             actor.level += playerPokemonAvg - enemyPokemonAvg;
 
-                            for (const statistic of statistics) {
-                                (actor as any)[statistic] = (actor as any)[statistic + "Normal"] =
-                                    this.MathDecider.compute("pokemonStatistic", actor, statistic);
+                            if (this.utilities.shouldEvolve(actor)) {
+		                        this.utilities.evolvePokemon(actor);
+	                        } else {
```

Comment:
I had gotten an error about the else at one point and did some fiddling, including moving it up a line, that fixed. It totally could've been the other stuff that fixed it but I didn't bother trying it again with the else on the next line. I wasn't sure what convention you wanted me to use, but I'll get on that.